src/components/blocks/admission/admission-grid.js

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { StaticQuery, graphql } from "gatsby"
+import { Row, Col } from "react-bootstrap";
+import PageContainer from 'components/shared/pageContainer'
+import styled from "styled-components"
+
+const yaml = require('js-yaml');
+
+const Background = styled(PageContainer.FullWidth)`
+  background: #F4F7FA;
+`
+const HeadingIcon = styled.i`
+  color: ${props => (props.iconColour ?? "#000000")};
+`
+
+const render = ({ field_yaml_map }) => {
+    let yamlMap;
+
+    try {
+      yamlMap = yaml.load(field_yaml_map);
+    } catch (e) {
+      console.log(e);
+      return null;
+    }
+    
+    return (
+      <Background>
+        <PageContainer.SiteContent>
+          <PageContainer.ContentArea>
+            {yamlMap.title && <h2 className="mt-4">{yamlMap.title}</h2>}
+            <Row className="row-cols-1 row-cols-md-3 g-4 mt-0">
+                {yamlMap.content.map(({title, body_html, icon, icon_color}, index) => 
+                    <Col key={`admission-supplementary-grid-${index}`} className="pe-4">
+                      <h3 className="h4 text-dark mt-0 d-flex align-items-start"><HeadingIcon aria-hidden={true} className={`${icon} pe-3`} iconColour={icon_color} /> {title}</h3>
+                      <div dangerouslySetInnerHTML={{__html: body_html}}></div>
+                    </Col>
+                )}
+            </Row>
+          </PageContainer.ContentArea>
+        </PageContainer.SiteContent>
+      </Background>
+)}
+
+
+const query = graphql`
+  query {
+    blockContentYamlBlock(field_yaml_id: {glob: "admission_supplementary_block"}) {
+      id
+      field_yaml_id
+      field_yaml_map
+    }
+  }
+`
+
+export default function AdmissionGrid() {
+  return <StaticQuery query={query} render={({blockContentYamlBlock}) => render(blockContentYamlBlock)} />
```

Comment:
`StaticQuery` is deprecated as per https://www.gatsbyjs.com/docs/how-to/querying-data/static-query/ - might it be possible to change this to use `useStaticQuery` instead? https://www.gatsbyjs.com/docs/how-to/querying-data/use-static-query/

Reply:
I like this idea. If Linda needs it super quick and you have time, feel free to implement the change on the branch, otherwise I'll add it after lunch (Still stuck in line 🥲)

Reply:
I'm out of the line! Will implement.

Reply:
Switched to useStaticQuery

Reply:
Works as expected